application/backend/src/main/resources/apps/typerefinery/components/forms/fields/button/clientlibs/functions.js

```diff
@@ -4,9 +4,10 @@ window.Typerefinery.Components.Forms = Typerefinery.Components.Forms || {};
 window.Typerefinery.Components.Forms.Button = Typerefinery.Components.Forms.Button || {};
 window.Typerefinery.Modal = Typerefinery.Modal || {};
 window.Typerefinery.Dropdown = Typerefinery.Dropdown || {};
+window.Typerefinery.ToggleComponent = Typerefinery.ToggleComponent || {};
 window.Typerefinery.Page.Theme = Typerefinery.Page.Theme || {};
 
-(function (ns, componentNs, modalNs, dropdownNs, themeNs, document, window) {
+(function (ns, componentNs, modalNs, dropdownNs, togglecomponentNs, themeNs, document, window) {
```

Comment:
Please follow Camel case for the naming conversions.

Reply:
Changed.